File: 1. Two Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        dict = {};
        for i in range(len(nums)):
            j = target-nums[i]
            if j in dict:
                return [dict[j],i]
            dict[nums[i]]=i

File: 1. Two Sum/1. Two Sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */

const twoSum = function (nums,target) {
    let obj = {};
    for(let i = nums.length - 1;i>=0;i--){
        let j = target-nums[i];
        if(typeof obj[j] !== 'undefined'){
            return [obj[j],i]
        }
        obj[nums[i]] = i
    }
    return '404 not found'
}
